kontakt-reservasjonsregister-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/krr/DSFClient.java

```diff
@@ -60,6 +61,10 @@ public DSFResource getDSFResource(String identifier, String token) throws KRRCli
             throw new KRRClientException("Error mapping payload to " + DSFResponse.class.getName(), e);
         }
 
-        return dsfResponse.getPersons().get(0);
+        if (dsfResponse.getPersons() == null || dsfResponse.getPersons().size() == 0) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use isEmpty() to check whether the collection is empty or not. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1155)